cds_ils/documents/api.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS document API."""
+
+from invenio_app_ils.documents.api import Document as ILSDocument
+from invenio_pidstore.resolver import Resolver
+
+DOCUMENT_LEGACY_PID_TYPE = "lrecid"
+
+
+class Document(ILSDocument):
```

Comment:
I find it a bit dangerous in terms of naming, it is very easy to miss from where the Document was imported, CDS-ILS or invenio-app-ils. I would name it `CdsDocument` to leave no doubts.

Reply:
The same happens with Patron and AnonymousPatron class, they should be changed too? Also since we are using the `record_class` from the config do we care from where the class was imported?